### ISSUE TITLE
Ensure inline links in `LaunchErrorDialog` are underlined

### ISSUE THREAD

**Issue by robertknight** [hypothesis/lms#4355]
The `LaunchErrorDialog` includes several messages with inline links, rendered using the `<Link>` component. These currently have no underline, so are not obviously links for users who can't distinguish red/black. Once the newest @hypothesis/frontend-shared package is released, we should update these links to have underlines.
